ephios/core/templates/core/disposition/fragment_participation.html

```diff
@@ -6,34 +6,40 @@
-                {{ form.instance.participant.icon }}
-                {{ form.instance.participant }}
-            </h6>
-            {% with form.instance|conflicting_participations as conflicts %}
-                {% if conflicts %}
-                    <span class="fa fa-exclamation-triangle text-danger"
-                          data-bs-toggle="tooltip"
-                          title="{% blocktranslate trimmed with participant=form.instance.participant conflicts=conflicts|join:", " %}
-                                     {{ participant }} already participates in {{ conflicts }} at the same time.
-                                 {% endblocktranslate %}"></span>
-                {% endif %}
-            {% endwith %}
-
-            {% if form.instance.participant.is_minor %}
-                <span class="badge bg-warning">{% translate "Minor" %}</span>
-            {% endif %}
-            {% for abbreviation in form.instance.participant.qualifications|qualifications_to_essential_abbreviations %}
-                <span class='badge bg-secondary text-dark'>{{ abbreviation }}</span>
-            {% endfor %}
+        <div class="col">
+            <div class="row">
+                <div class="col pt-1 pb-2">
+                    <h6 class="d-inline">
+                        {{ form.instance.participant.icon }}
+                        {{ form.instance.participant }}
+                    </h6>
+                    {% with form.instance|conflicting_participations as conflicts %}
+                        {% if conflicts %}
+                            <span class="fa fa-exclamation-triangle text-danger"
+                                  data-bs-toggle="tooltip"
+                                  title="{% blocktranslate trimmed with participant=form.instance.participant conflicts=conflicts|join:", " %}
+                                             {{ participant }} already participates in {{ conflicts }} at the same time.
+                                         {% endblocktranslate %}"></span>
+                        {% endif %}
+                    {% endwith %}
 
-            {% block participation_detail %}
-            {% endblock %}
+                    {% if form.instance.participant.is_minor %}
+                        <span class="badge bg-warning">{% translate "Minor" %}</span>
+                    {% endif %}
+                    {% for abbreviation in form.instance.participant.qualifications|qualifications_to_essential_abbreviations %}
+                        <span class='badge bg-secondary text-dark'>{{ abbreviation }}</span>
+                    {% endfor %}
+                    {% block participation_detail %}
+                    {% endblock %}
+                </div>
+            </div>
         </div>
         <div class="col-auto d-flex justify-content-between align-items-center">
-            {% block participation_controls %}
-            {% endblock %}
+            {% if form.can_delete %}
```

Comment:
could be a bit weird for users that the controls now swapped their place, but probably fine to gain consistency

Reply:
I don't think it's a commonly used button anyway